openproblems/tasks/spatial_decomposition/methods/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .nmfreg import nmfreg
+from .nnls import nnls_scipy
+from .nusvr import nusvr_sklearn
+from .random import random_proportion_assignment
+from .rctd import rctd
+from .seuratv3 import seuratv3
+from .stereoscope import stereoscope_raw
+from .vanillanmf import nmf
```

Comment:
For each of these new methods, you will need to add a `test=False` keyword argument. This is new as of the last update from `main`.

Reply:
done